sdp/processors/modify_manifest/data_to_data.py

```diff
@@ -13,15 +13,99 @@
 # limitations under the License.
 
 import collections
+import os
 import re
 from typing import Dict, List
 
+import soundfile
+from sox import Transformer
+
 from sdp.logging import logger
 from sdp.processors.base_processor import BaseParallelProcessor, DataEntry
 from sdp.utils.edit_spaces import add_start_end_spaces, remove_extra_spaces
 from sdp.utils.get_diff import get_diff_with_subs_grouped
 
 
+class GetAudioDuration(BaseParallelProcessor):
+    """
+    Processor to count audio duration using audio file path from input_field
+
+    Args:
+        audio_filepath_field (str): where to get path to wav file.
+        duration_field (str): where to put to audio duration.
+    Returns:
+        All the same fields as in the input manifest plus output_field
```

Comment:
"All the same fields as in the input manifest plus ``duration_field``"?

Reply:
done